packages/radix-ui-themes/src/theme.tsx

```diff
@@ -82,61 +82,27 @@ const ThemeRoot = React.forwardRef<ThemeImplElement, ThemeRootProps>((props, for
   const [scaling, setScaling] = React.useState(scalingProp);
   React.useEffect(() => setScaling(scalingProp), [scalingProp]);
 
-  // Initial appearance on page load when `appearance` is explicitly set to `light` or `dark`
-  const ExplicitRootAppearanceScript = React.memo(
-    ({ appearance }: { appearance: Exclude<ThemeOptions['appearance'], 'inherit'> }) => (
-      <script
-        dangerouslySetInnerHTML={{
-          __html: `!(function(){try{var d=document.documentElement,c=d.classList;c.remove('light','dark');d.style.colorScheme='${appearance}';c.add('${appearance}');}catch(e){}})();`,
-        }}
-      ></script>
-    ),
-    () => true // Never re-render
-  );
-  ExplicitRootAppearanceScript.displayName = 'ExplicitRootAppearanceScript';
-
-  // Client-side only changes when `appearance` prop is changed while developing
-  React.useEffect(() => updateThemeAppearanceClass(appearanceProp), [appearanceProp]);
-
-  const resolvedGrayColor = grayColor === 'auto' ? getMatchingGrayColor(accentColor) : grayColor;
-
   return (
-    <>
-      {appearance !== 'inherit' && <ExplicitRootAppearanceScript appearance={appearance} />}
-
-      {hasBackground && (
-        <style
-          dangerouslySetInnerHTML={{
-            __html: `
-:root, .light, .light-theme { --color-page-background: white; }
-.dark, .dark-theme { --color-page-background: var(--${resolvedGrayColor}-1); }
-body { background-color: var(--color-page-background); }
-`,
-          }}
-        />
-      )}
```

Comment:
Removing this is made possible by:
- The root `Theme` covers `100vh` and gets the background color, so we don't care about the body background anymore
- Since we don't care about the body background anymore, we don't need the theme classes on the html/body
- Now we don't need to inject the script and styles, suppress the html hydration warning, and to sync root appearance class in dev with the prop value